### PR TITLE
Issue #128: iOS - UI - Detail Screen: Scaffold initial screen with elements

### PR DESCRIPTION
**Pull request by DevHarleyA**
### Changes
- Implements issue #128 - Scaffold initial screen with elements
- Add comments for related task

### Screenshot
Screenshot 2024-02-11 at 1 35 50 PM

Review Request: @clc80 @devanshimodha

### CODE REVIEW

File: coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift [hunk 1]
```diff
@@ -8,8 +8,129 @@
 import SwiftUI
 
 struct DetailsScreen: View {
+    @ObservedObject var viewModel = DetailsViewModel()
+    @State private var taskText: String = ""
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(spacing: 5) {
+            Spacer()
```
Comment (clc80) [inline coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift:16]:
```suggestion
            
```
Can we get rid of this spacer,  it puts too much room in the top.

**Review by clc80 — CHANGES_REQUESTED** [review]
This is really good. Just one nit pick.

File: coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift (hunk 1 above)
Comment (DevHarleyA) [reply coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift]: Yes, removed!

Comment (DevHarleyA) [reply coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift]: I updated the screenshot as well in the description

**Review by clc80 — APPROVED** [review]
Thank you. Looks better.

**Review by devanshimodha — APPROVED** [review]

**Pull request merged**